packages/themes/src/tools/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export * from './modifiers';
```

Comment:
Would it be helpful to have a single `tools.js` file instead of doing the directory plus entrypoint plus modifiers files?

Reply:
Yeah, makes more sense.